src/Dialog/DialogTitle.js

```diff
@@ -42,7 +42,7 @@ export type Props = {
   className?: string,
   /**
    * If `true`, the children won't be wrapped by a typography component.
-   * For instance, that can be useful to can render an h4 instead of a
+   * For instance, that can be useful to render a h4 instead of the default h2.
```

Comment:
Looks good, except the original "an" was correct, at least in American English.

Reply:
And English English. 😄 I would also change 'that' to 'this'.

Reply:
Also, please cloud you update the base commit title to match the PR title (as it is now) - it keeps make preparing the CHANGELOG much easier! Thanks.

Reply:
> Also, please cloud you update the base commit title to match the PR title (as it is now) - it keeps make preparing the CHANGELOG much easier! Thanks.

No neeed. We can squash & rebase. We can put whatever we prefer as admin.